fe/fe-core/src/test/java/com/starrocks/analysis/PauseRoutineLoadStmtTest.java

```diff
@@ -0,0 +1,33 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.utframe.UtFrameUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class PauseRoutineLoadStmtTest {
+
+    private ConnectContext ctx;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
If setUp method is empty, I think we needn't it.

Reply:
[remove setup method because this method is empty](https://github.com/StarRocks/starrocks/pull/8985/commits/16ced90a49487c69612aa78fa8041582294c1ae0)